install-devstack-xen.sh

```diff
@@ -122,6 +144,10 @@ while getopts ":t:d:fnl:j:e:s:" flag; do
             SUPP_PACK_URL="$OPTARG"
             REMAINING_OPTIONS=$(expr "$REMAINING_OPTIONS" - 1)
             ;;
+        u)
+            UBUNTU_REPO_URL="$OPTARG"
```

Comment:
This is not the must part, but in our local test, we can use our internal repo instead of the public repo which can make downloading faster